test/clt-tests/buddy/test-fuzzy-trailing-semicolon.rec

```diff
@@ -0,0 +1,194 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -P9306 -h0 -e "drop table if exists t; create table t(f text)"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P9306 -e "INSERT INTO t(id, f) VALUES (1, 'ящик'), (2, 'коробка');"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P9306 -e "SELECT * FROM t;"
```

Comment:
I think we should prove that we have data in another test, this tests is about that semicolons works